nylas/client/client.py

```diff
@@ -278,6 +316,67 @@ def availability(
             "start_time": start_time,
             "end_time": end_time,
             "free_busy": free_busy or [],
+            "open_hours": open_hours or [],
+        }
+        resp = self.session.post(url, json=data)
+        _validate(resp)
+        return resp.json()
+
+    def consecutive_availability(
+        self,
+        emails,
+        duration,
+        interval,
+        start_at,
+        end_at,
+        buffer=None,
+        free_busy=None,
+        open_hours=None,
+    ):
+        if isinstance(emails, six.string_types):
+            emails = [[emails]]
```

Comment:
API will throw an error if you pass in a 1D array too, should there be a check for that?

Reply:
Sounds good, check has been added!